Orange/widgets/unsupervised/owhierarchicalclustering.py

```diff
@@ -1174,6 +1189,7 @@ def commit(self):
             return
 
         selection = self.dendrogram.selected_nodes()
+        self.selected_trees = list(map(hash, selection))
```

Comment:
Hashes should not be considered as stable ids suitable for persistence (neither are they guaranteed unique).

Reply:
I am aware that they are not completely unique however I think they are still a suitable as ids because the probability of having a collision is very small. Do you recommend any other parameter I could use as ids?